### DISCUSSION TITLE
[ZK Stack] - Failure to claim bridged assets on ZK Portal

### DISCUSSION THREAD

**Discussion by eXterkTi** [zkSync-Community-Hub/zksync-developers#711] [General] [answered]
### Module Affected

- [ ] System contracts (era-system-contracts)
- [ ] Smart contracts (era-contracts)
- [ ] zkEVM (era-zk_evm)
- [ ] Server (zksync-era)
- [ ] Prover (era-boojum)
- [ ] Fee withdrawer (era-fee-withdrawer)
- [ ] Block Explorer (clock-explorer)
- [X] Portal - Wallet + Bridge (dapp-portal)
- [ ] Other (please specify below)

### Rust Version

1.75.0

### Operating System

- [ ] Ubuntu
- [X] MacOS
- [ ] Windows
- [ ] Other (Please specify below)

### Issue Description

I was trying to claim my bridged asset (ETH) on Ethereum mainnet from ZKsync on ZK Portal, I got fee estimation error that preventing me from claiming it. See below for message body:

Fee estimation error: The contract function "finalizeWithdrawal" reverted with the following reason:
xx

Contract Call:
address: 0xd7f9f54194c633f36ccd5f3da84ad4a1c38cb2cb

<img width="1512" alt="image" src="https://github.com/user-attachments/assets/dbc3a644-6c3c-4e72-a352-7b51beb36e93">

### Expected Behavior

Getting Fee estimation error while claiming asset on the dest chain (ETH)

Error Message:
Fee estimation error: The contract function "finalizeWithdrawal" reverted with the following reason:
xx

Contract Call:
address: 0xd7f9f54194c633f36ccd5f3da84ad4a1c38cb2cb

### Current Behavior

Unable to claim assets on dest chain.

### Prior Research

- [X] Yes, I searched for similar issues and did not find any matching ones.

**Comment by alhenaai** [comment]
It appears you are encountering a fee estimation error when trying to claim your bridged assets from ZKsync to the Ethereum mainnet using the ZK Portal. This issue typically arises due to the smart contract function `finalizeWithdrawal` reverting for some reason.

Here are a few steps you can take to potentially resolve this issue:

1. **Check Gas Prices:** Ensure that the gas prices on the Ethereum network are not too high at the moment, as this can affect transaction costs and execution.
2. **Retry the Transaction:** Sometimes, simply retrying the transaction when the network is less congested may resolve the issue.
3. **Check Contract Address:** Verify that the contract address `0xd7f9f54194c633f36ccd5f3da84ad4a1c38cb2cb` is correct and that there are no issues reported with this contract.
4. **Portal Updates:** Ensure that your Portal interface is up to date and that there are no ongoing issues with the Portal itself that could be causing this error.

If the problem persists, I recommend reaching out to the ZKsync support team directly or checking if there are any updates or notices regarding this issue on the official [ZKsync GitHub repository](https://github.com/matter-labs/zksync) or community forums.

Your feedback helps us serve you better. If this fully addresses your question, please give it a thumbs up. If it doesn't, please give it a thumbs down. This will inform the community and team so we can improve our support.

**Comment by bxpana** [comment, marked as answer]
Hey @eXterkTi sorry to hear you're running into issues with claiming. This is a known issue for the ZKsync Portal, but luckily we have other portals run my the community that do work. You can use https://portal.txsync.io/ or https://syncswap.xyz/move to successfully claim your withdrawal.

**Comment by eXterkTi** [comment]
Thanks for the tips. I've managed to claim it via community portals listed
above.

On Tue, Oct 1, 2024 at 3:27 PM bxpana ***@***.***> wrote:

> Hey @eXterkTi <https://github.com/eXterkTi> sorry to hear you're running
> into issues with claiming. This is a known issue for the ZKsync Portal, but
> luckily we have other portals run my the community that do work. You can
> use https://portal.txsync.io/ or https://syncswap.xyz/move to
> successfully claim your withdrawal.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/zkSync-Community-Hub/zksync-developers/discussions/711#discussioncomment-10812935>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHZ5XIQPQNHORTFJ4IQTOWLZZLZR7AVCNFSM6AAAAABPFWD33KVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAOBRGI4TGNI>
> .
> You are receiving this because you were mentioned.Message ID:
> <zkSync-Community-Hub/zksync-developers/repo-discussions/711/comments/10812935
> @github.com>
>


-- 
*Edrick Y Guan*
***@***.***

**Reply by bxpana** [reply]
Glad that worked for you!
